src/core/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,133 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../utils/hooks/useTheme';
+import { CommonProps } from '../utils/props';
+import SvgChevronRight from '@itwin/itwinui-icons-react/cjs/icons/ChevronRight';
+import { IconButton } from '../Buttons';
+import { Tooltip } from '../Tooltip';
+import '@itwin/itwinui-css/css/side-navigation.css';
+
+export type SidebarProps = {
+  /**
+   * Buttons shown in the top portion of sidebar.
+   * Recommended to use `SidebarButton` components.
+   */
+  mainItems: React.ReactNode[];
+  /**
+   * Buttons shown at the bottom of sidebar.
+   */
+  secondaryItems?: React.ReactNode[];
+  /**
+   * Control the visibility of "expander" icon button.
+   * @default 'top'
+   */
+  expanderVisibility?: 'top' | 'bottom' | 'hidden';
```

Comment:
Visibility or Position...

Reply:
"position: hidden" would not make sense

Reply:
True, but visibility: "top" also is weird..

Reply:
it made sense when I thought to myself: it is "visible" on the top 😄

Reply:
Yeah I missed this one but it really looks odd, position would be better here. And maybe it would sound better if `hidden` would be changed with `none`?

Reply:
How about `expanderPlacement`? to be consistent with `placement` from popover